modules/ftui/ftui.helper.js

```diff
@@ -227,6 +227,61 @@ export function durationFromSeconds(time) {
   return ret;
 }
 
+export function agoTillFormat(ms, format, mode='lower') {
+  var x = ms / 1000;
+
+  var SECONDS = Math.floor(x);
```

Comment:
var is deprecated. Please use let or const instead.

Reply:
'var' replaced by 'const' or 'let'.
'Math.floor' replaced by '~~'.